### DISCUSSION TITLE
find MCS with minimal lib

### DISCUSSION THREAD

**Discussion by iwatobipen** [rdkit/rdkit#4971] [Q&A] [answered]
Dear all,

Recently I'm interested in rdkit cffi. And I have one question.
Is it possible to find MCS with rdkit minimal lib ?
Find MCS is common task for medchem SAR analysis so I would like to know how to do it with the library.
Thanks,

Taka

**Comment by greglandrum** [comment, marked as answer]
Hi Taka. 
This isn't currently available in the minimal lib, but it could be. :-)

If it's something you're interested in, please create a feature request in the issues tab.

**Reply by iwatobipen** [reply]
Thanks for your prompt reply! It sounds nice.
